regression-tests.dnsdist/test_Carbon.py

```diff
@@ -64,6 +77,30 @@ def startResponders(cls):
         cls._CarbonResponder2.setDaemon(True)
         cls._CarbonResponder2.start()
 
+    @classmethod
```

Comment:
This is a quite complicated way to have some servers up, perhaps you could just force them up in the configuration, so you don't have to actually start responder threads? The more responder threads we start the more issue we potentially have later with another test reusing the same port too quickly. Something like:
```
server = newServer(...)
server:setDown()
server = newServer(...)
server:setUp()
server = newServer(...)
server:setUp
```